### PR TITLE
[Dart] use LOGGER.debug instead of System.out.println

### PR DESCRIPTION
**Pull request by wing328**
- use LOGGER.debug instead of System.out.println
- minor improvement to code format

### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (5.3.0), `6.0.x`
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

cc @jaumard (2018/09) @josh-burton (2019/12) @amondnet (2019/12) @sbu-WBT (2020/12) @kuhnroyal (2020/12) @agilob (2020/12) @ahmednfwela (2021/08)

### CODE REVIEW

**Comment by kuhnroyal** [comment]
This is my bad...

**Review by kuhnroyal — APPROVED** [review]

**Comment by wing328** [comment]
No worries. No one complains. I spotted it when I was updating the samples.

File: modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java [hunk 1]
```diff
@@ -427,10 +427,11 @@ private void addBuiltValueSerializerImport(String type) {
 
     /**
      * Adds the serializer to the global list of custom built_value serializers.
+     *
      * @param serializer
      */
     private void addBuiltValueSerializer(BuiltValueSerializer serializer) {
-        System.out.println("######## Add serializer!");
+        LOGGER.debug("######## Add serializer!");
```
Comment (agilob) [inline modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java:434]: This log message doesn't say anything valuable, I'm for removing it.

**Comment by agilob** [comment]
@wing328 check my other PR, added test case to prevent this happening again #11130

**Comment by wing328** [comment]
Merged via https://github.com/OpenAPITools/openapi-generator/pull/11130 instead.

**Pull request closed without merging**